data/fro/Decrecy_1997_yPonthus/metadata.txt

```diff
@@ -0,0 +1,216 @@
+publicationType : "book"
+publicationTitle : "Le roman de Ponthus et Sidoine. Édition critique de Marie-Claude de Crécy"
+publicationDate : "1997"
+publicationPlaces:
+    - publicationPlace: "Genève"
+publicationSeries : ""
+publicationNum: ""
+publicationStemmaNum: "2"
+publicationAuthors:
+    - publicationAuthor: "Marie-Claude de Crécy"
+publicationPage : "LVIX"
+publicationLink : ""
+workTitle : "Ponthus et Sidoine"
+workViaf : ""
+workOrigDate : "at the end of the 14th or the beginning of the 15th c."
+workOrigPlace : "Bretagne?"
+workAuthors:
+    - workAuthor: "Anonymous"
+      workAuthorViaf: ""
+workGenre : "roman  en prose"
+workLangCode : "frm"
+stemmaType : "reconstructed"
+drawnStemma : "true"
+contam : "yes"
+extraStemmContam : "no"
+rootType : "archetype"
+completeWits : "included"
+sourceText : "excluded"
+derivatives : "excluded"
+contributors:
+    - contributor: "Benedetta Salvati"
+      contributorORCID: ""
+note : ""
+wits:
+    - witSigla: "G"
```

Comment:
Siglum "G" is repeated and that is not allowed in our model. (see below line 189)